src/control/server/README.md

```diff
@@ -163,205 +163,65 @@ In order to run the tool to perform administrative tasks, build and run the `dao
 
 See `daos_shell --help` for usage and [here](../dmg) for package details.
 
-## NVMe management capabilities
+## Storage management
 
-Operations on NVMe SSD devices are performed using [go-spdk bindings](./go-spdk/README.md) to issue commands over the SPDK framework.
+The DAOS data plane utilises two forms of non-volatile storage, storage class memory (SCM) in the form of persistent memory modules and NVMe in the form of high-performance SSDs.
```

Comment:
I think it would be nice to also have the role of control plane described here, for providing a more complete picture.